deploy/operator.yaml

```
@@ -25,6 +25,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: APP_NAMESPACES
+                value: "unifiedpush"
```

```suggestion
              value: "unifiedpush"
```
so it looks like the test is failing because of these 2 extra spaces